requirements/dev.txt

```diff
@@ -0,0 +1,11 @@
+-e git+https://github.com/django-mptt/django-mptt#egg=django-mptt
```

Comment:
-e is doing git repo for editing, without it doing just snapshot. Should be faster.